timeseries/src/autogluon/timeseries/metrics/__init__.py

```diff
@@ -0,0 +1,56 @@
+import json
+from typing import Type, Union
+
+from .abstract import TimeSeriesScorer
+from .point import MAE, MAPE, MASE, MSE, RMSE, RMSSE, WAPE, sMAPE
+from .quantile import WQL
+
+__all__ = [
+    "MAE",
+    "MAPE",
+    "MASE",
+    "sMAPE",
+    "MSE",
+    "RMSE",
+    "RMSSE",
+    "WAPE",
+    "WQL",
+]
+
+DEFAULT_METRIC = "WQL"
+
+AVAILABLE_METRICS = {
+    "MASE": MASE,
+    "MAPE": MAPE,
+    "SMAPE": sMAPE,
+    "RMSE": RMSE,
+    "RMSSE": RMSSE,
+    "WAPE": WAPE,
+    "WQL": WQL,
+    # Exist for compatibility
+    "MSE": MSE,
+    "MAE": MAE,
+}
+
+
+def check_get_evaluation_metric(
+    eval_metric: Union[str, TimeSeriesScorer, Type[TimeSeriesScorer], None] = None,
+) -> TimeSeriesScorer:
```

Comment:
We could also support passing a `Scorer` object from `autogluon.tabular` here, and automatically convert it into a `PointForecastMetric` - but this could make the API even more complex.

Reply:
Good point, I agree.